### PR TITLE
DTFS2-4412 Assign UKEF IDs

### PR DESCRIPTION
**Pull request by eddieleeper**

### CODE REVIEW

**Review by pricea1 — DISMISSED** [review]
LGTM

File: portal-api/api-tests/v1/gef/application.api-test.js [hunk 1]
```diff
@@ -293,6 +317,31 @@ describe(baseUrl, () => {
         expect(secondPutResponse.body.submissionDate).toEqual(initialSubmissionDate);
       });
 
+      it('adds a ukefFacilityId to each issues facility associated with the application', async () => {
```
Comment (ttbarnes) [inline portal-api/api-tests/v1/gef/application.api-test.js:320]: issueD* 😛

File: portal-api/api-tests/v1/gef/application.api-test.js [hunk 2]
```diff
@@ -293,6 +317,31 @@ describe(baseUrl, () => {
...
+        const facilityId = createFacilityResponse.body.details._id;
+
+        // check that the facility does not already have submittedAsIssuedDate
+        expect(createFacilityResponse.body.details.ukefFacilityId).toEqual(null);
+
+        // change deal status to submitted
+        const putResponse = await as(aMaker).put({ status: 'SUBMITTED_TO_UKEF' }).to(`${baseUrl}/status/${body._id}`);
+        expect(putResponse.status).toEqual(200);
+
+        // check that the facility has updated submittedAsIssuedDate
```
Comment (ttbarnes) [inline portal-api/api-tests/v1/gef/application.api-test.js:339]: need to replace submittedAsIssuedDate with ukefFacilityId

**Review by ttbarnes — DISMISSED** [review]
Nice one 🚀

**Review by ttbarnes — APPROVED** [review]

**Pull request merged**